# Name of the domain bridge, used for node naming and logging
name: turtle_bridge # TODO: Problem in domain 1, multiple nodes named turtle_bridge1
from_domain: 1
to_domain: 2
topics:
  # Bridge "/target" topic from domain ID 1 to domain ID 2
  target:
    type: geometry_msgs/msg/PointStamped

  distanceToTarget:
    type: test_multi_id_interfaces/msg/DistanceToTarget
    reversed: true

  assignedRobot:
    type: std_msgs/msg/Int8